enhancements/monitoring/alerting-consistency.md

```diff
@@ -99,37 +140,39 @@ alerts so they fire before the situation becomes irrecoverable. Expect users to
 be notified of a critical alert within a short period of time after it fires so
 they can respond with corrective action quickly.
 
-Some disaster situations are:
-* loss or impending loss of etcd-quorum
-* etcd corruption
-* inability to route application traffic externally or internally
-(data plane disruption)
-* inability to start/restart any pod other than capacity.  EG, if the API server
-were to restart or need to be rescheduled, would it be able to start again?
-
-In other words, critical alerts are something that require someone to get out
-of bed in the middle of the night and fix something **right now** or they will
-be faced with a disaster.
-
-An example of something that is NOT a critical alert:
-* MCO and/or related components are completely dead/crash looping.
-
-Even though the above is quite significant, there is no risk of immediate loss
-of the cluster or running applications.
-
-You might be thinking to yourself: "But my component is **super** important, and
-if it's not working, the user can't do X, which user definitely will want."  All
-that is probably true, but that doesn't make your alert "critical" it just makes
-it worthy of an alert.
-
-The group of critical alerts should be small, very well defined,
-highly documented, polished and with a high bar set for entry. This includes a
+Example critical alert: [KubeAPIDown][3]
+
+```yaml
+- alert: KubeAPIDown
+  annotations:
+    summary: Target disappeared from Prometheus target discovery.
+    description: KubeAPI has disappeared from Prometheus target discovery.
+    runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/KubeAPIDown.md
+  expr: |
+    absent(up{job="apiserver"} == 1)
+  for: 15m
+  labels:
+    severity: critical
+```
+
+This alert fires if no Kubernetes API server instance has reported metrics
+successfully in the last 15 minutes.  This is a clear example of a critical
+control-plane issue that represents a threat to the operability of the cluster
+as a whole, and likely warrants paging someone.  The alert has clear summary and
+description annotations, and it links to a runbook with information on
+investigating and resolving the issue.
+
+The group of critical alerts should be small, very well defined, highly
+documented, polished and with a high bar set for entry.  This includes a
 mandatory review of a proposed critical alert by the Red Hat SRE team.
```

Comment:
We should add details on how and where this happens? How does someone contact the SRE team for this review?